website/src/middleware.js

```diff
@@ -198,7 +198,8 @@ module.exports.locals = function(req, res, next) {
     Content: shared.content,
     mods: require('./models/user').mods,
     tavern: tavern, // for world boss
-    worldDmg: (tavern && tavern.quest && tavern.quest.extra && tavern.quest.extra.worldDmg) || {}
+    worldDmg: (tavern && tavern.quest && tavern.quest.extra && tavern.quest.extra.worldDmg) || {},
+    _: _
```

Comment:
It's for this: https://github.com/SabreCat/habitrpg/blob/pet-skins/website/views/options/profile.jade#L7

That's what @lefnire and I came up with after banging around quite a bit, but if there's a smoother way to get this logic in without needing lodash as middleware, that may be preferable.